### ISSUE TITLE
Parse error for unsigned right shift assignment

### ISSUE THREAD

**Issue by elasmojs** [boa-dev/boa#621]
Hi,

Following code results in parse error;
```js
let a = 5; //  00000000000000000000000000000101

a >>>= 2;  //  00000000000000000000000000000001
console.log(a);
// expected output: 1

let b = -5; // -00000000000000000000000000000101

b >>>= 2;   //  00111111111111111111111111111110
console.log(b);
// expected output: 1073741822
```
thanks

**Comment by HalidOdat** [comment]
This is Fixed on master branch

**Comment by Razican** [comment]
Actually, the `>>>=` operator is not yet implemented. I'm re-opening this.

**Comment by HalidOdat** [comment]
> Actually, the `>>>=` and `>>>` operators are not yet implemented. I'm re-opening this.

The `>>>=` is not implemented but `>>>` is (in #520). `>>>=` should be easy to implement.

**Comment by arpit-saxena** [comment]
I would like to work on this.
